about/policy/metadata/index.md

```diff
@@ -6,109 +6,57 @@ tags:
-
-Let's take a closer look at the basic elements of SGID metadata. We like to think of these as our minimum requirements.
-
-### Basic SGID Metadata
-{: .text-left}
+Effective metadata helps people understand the source of a dataset, how up-to-date it is, and any of its inherent restrictions or limitations. Listed below are the minimum metadata items that should be present in any SGID datasets. These correspond to the Esri [Item Description](https://pro.arcgis.com/en/pro-app/latest/help/metadata/view-and-edit-metadata.htm) metadata style and the metadata displayed on an item's ArcGIS Online page.
+## Basic SGID Metadata
 
 #### Title
 {: .text-left}
 
 The title should match the dataset name. If it's a statewide dataset, the title should begin with the word "Utah".
 
-`example: Utah County Boundaries`
+Example: `Utah County Boundaries`
 
 #### Tags
 {: .text-left}
 
-Tags are included in data searches, so at minimum, each SGID dataset should include the stewarding agency (ie: UGRC), SGID, and the appropriate ISO category tag. We also recommend adding any additional, relevant tags that might help with data discoverability. However, there is no need to include words that are contained in the layer's title (these will be omitted).
+Tags are used by ArcGIS Online in its search algorithms and can be used in Open Data to group data. At a minimum, each SGID dataset's tags should include the stewarding agency (ie: UGRC), "SGID", and the appropriate category name. We also recommend adding any additional, relevant tags that might help with data discoverability. However, there is no need to include words that are contained in the layer's title (these will be omitted).
 
 All words in tags should be title-cased except for articles (“of”, “the”, “and”, etc.) or abbreviations. Each tag should be separated by a comma. Please refer to [Using tags effectively](https://www.esri.com/arcgis-blog/products/arcgis-online/data-management/using-tags-effectively/) for more ideas and information.
 
-`example: UGRC, SGID, Recreation, Paths, Biking, Hiking, Horseback Riding`
+Example: `UGRC, SGID, Recreation, Paths, Biking, Hiking, Horseback Riding`
 
 #### Summary
 {: .text-left}
 
-The summary should summarize the description in approximately 2 to 3 sentences.
+The summary should _briefly_ explain the dataset in under 250 characters (an ArcGIS Online limitation). It is primarily used when reviewing ArcGIS Online search results. The summary may end up being very similar to the dataset name for some datasets.
 
-The summary should address the questions presented above in the [summarize your data](#summarize-your-data) section.
+Example: `Utah public library and bookmobile locations`
```

Comment:
**question:** should there be a link to the open data about page for this data set, to show its summary? (https://opendata.gis.utah.gov/datasets/utah::utah-public-libraries-2/about)

Reply:
I'm going to say no- the example text here is self-contained and won't break if that layer is updated, disappears, etc.